heron/proto/scheduler.proto

```diff
@@ -28,6 +29,11 @@ message RestartTopologyRequest {
   required int32 container_index = 2;
 }
 
+message UpdateTopologyRequest {
+  required heron.proto.system.PackingPlan currentPackingPlan = 1;
+  required heron.proto.system.PackingPlan proposedPackingPlan = 2;
+}
+
```

Comment:
This is send from the [RuntimeManagerRunner](https://github.com/twitter/heron/pull/1218/files#diff-175aa5efeb9ca85f99dfb52dead01fa7R175) to the [Scheduler](https://github.com/twitter/heron/compare/master...billonahill:billg/scaling_proto#diff-4a29bbb233b3d74c8c379edb6cf37487R122) to invoke the scaling event. 

Look at the rest of scheduler.proto to get a sense for how request/responses are implemented. Each request has it's own type, but all responses use the common SchedulerResponse that includes a status code indicating success.